README.md

```diff
@@ -1,21 +1,75 @@
 # Create an Accordion Component in a Blazor Server Application
 
-This is a quick-start project that helps you create Blazor Accordion provided by Syncfusion. This example explains how to create a Blazor application and add the Syncfusion Blazor package. It also shows how to add the Syncfusion Blazor Accordion component to a Blazor server application and demonstrates prominent features like expand modes and customizing Accordion UI using templates.
-
-Watch a How-to Video: https://www.syncfusion.com/tutorial-videos/blazor/accordion?title=create-an-accordion-component-in-a-blazor-server-application
-
-Documentation: https://blazor.syncfusion.com/documentation/accordion/getting-started
-
-Online example: https://blazor.syncfusion.com/demos/accordion/default-functionalities
+The [Blazor Accordion](https://www.syncfusion.com/blazor-components/blazor-accordion?utm_source=github&utm_medium=listing&utm_campaign=blazor-navigations-github-samples) is a container-based component with vertically collapsible panels (vertical accordion) and stacked headers that expand or collapse one or more panels at a time within the available space. This is a quick-start project that helps you create Blazor Accordion provided by Syncfusion. This example explains how to create a Blazor application and add the Syncfusion Blazor package. It also shows how to add the Syncfusion Blazor Accordion component to a Blazor server application and demonstrates prominent features like expand modes and customizing Accordion UI using templates.
```

Comment:
Can you check the UTM parameter in all places?
![image](https://user-images.githubusercontent.com/61148953/161201085-f2a05d97-dc98-4393-ae75-c33be2839b25.png)